app/controllers/tech_blog_controller.rb

```diff
@@ -11,11 +19,51 @@ def visits_per_technology
     Builders::BlogMetricChart::TechnologyVisits.call
   end
 
-  def visits_growth_mom
-    Builders::BlogMetricChart::TechnologyVisitsGrowthMom.call
+  def year_to_date_visits
+    Builders::ChartSummary::YearToDateBlogVisits.call.to_i
```

Comment:
Can `YearToDateBlogVisits` return an Integer?
It would uniform it with the other services

Reply:
Absolutely yes!